apps/web/src/components/milestone-logs/recruitment/MilestoneForm.tsx

```diff
@@ -89,8 +89,13 @@ export const MilestoneForm = <T extends MilestoneFormValues>(props: MilestoneFor
     }
   };
 
-  const handleOutcomeType = (group: string) => {
+  const handleOutcomeType = (group: string, setFieldValue: FormikHelpers<T>['setFieldValue']) => {
     const outcomeGroup = Object.values(OutcomeGroups).find(({ value }) => value === group);
+
+    if (outcomeGroup?.value === `${STATUS.REFERRAL_ACKNOWLEDGED}`) {
+      const milestoneId = milestones.find(s => s.status == outcomeGroup?.value);
+      setFieldValue('status', milestoneId?.id);
```

Comment:
this line is to cancel out the validation for milestone/ status, since I have nothing to conditionally validate it with in dto